src/pages/_app.tsx

```diff
@@ -1,15 +1,30 @@
 import 'styles/index.scss'
 import '../../public/vendor/fontawesome-pro-5.15.1-web/css/all.min.css'
-import type { AppProps } from 'next/app'
 import Head from 'next/head'
+import type { AppProps } from 'next/app'
 import { useRouter } from 'next/router'
+import { ReactNode } from 'react'
```

Comment:
can we use `import type` on these lines to indicate we are just importing types here?